test/Libraries/RevitIntegrationTests/DocumentTests.cs

```diff
@@ -74,5 +74,51 @@ public void WhenActiveDocumentResetIsRequiredVisualizationsAreCleared()
         {
             Assert.Inconclusive("Cannot test. API required for allowing closing all docs.");
         }
+
+        [Test]
+        [TestModel(@".\Document\LocalModel\Project1_LocalFile.rvt")]
+        public void CanGetWorksharingModelPath()
+        {
+            // Arrange
+            string samplePath = Path.Combine(workingDirectory, @".\Document\canGetWorksharingModelPath.dyn");
+            string testPath = Path.GetFullPath(samplePath);
+            string expectedWorksharingFilePath = @"DynamoRevit\test\System\Document\CentralModel";
+            bool expectedIsCloudPathResult = false;
+
+            // Act
+            ViewModel.OpenCommand.Execute(testPath);
+            RunCurrentModel();
+            string resultWorksharingPath = GetPreviewValue("3f5e9a8cb7344c52a3c4937455ee68b1") as string;
+            var resultIsCloudPath = GetPreviewValue("1b62b04935b84f58a31bf45efe48955d");
+
+            bool worksharingPathIsCorrect = resultWorksharingPath.Contains(expectedWorksharingFilePath);
```

Comment:
this is a bit redundant, just move things to the right of `=` to the assert below